dev/tasks/macros.jinja

```diff
@@ -25,9 +25,9 @@ on:
       - "*-github-*"
 {% endmacro %}
 
-{%- macro github_checkout_arrow(fetch_depth=1, submodules="recursive") -%}
+{%- macro github_checkout_arrow(fetch_depth=1, submodules="recursive", action_v="v4") -%}
   - name: Checkout Arrow
-    uses: actions/checkout@v4
+    uses: actions/checkout{{ action_v }}
```

Comment:
isn't the `@` missing? I would probably also add the v and only pass the `action_version=4`
```suggestion
    uses: actions/checkout@v{{ action_version }}
```